.github/workflows/shellcheck.yml

```diff
@@ -7,6 +7,7 @@ name: Shellcheck
 # is consistent with the one of developers.
 
 on:
+  merge_group:
```

Comment:
Same as @smelc 's question for `actionlint.yaml`. Seems redundant for merge queue.